tao_compiler/decoupling/BUILD

```diff
@@ -1,7 +1,7 @@
 licenses(["notice"])  # Apache 2.0
 
 load("//tensorflow:tensorflow.bzl", "tf_cc_binary", "tf_cc_test")
-load("//tensorflow/compiler/mlir/disc:disc.bzl", "if_platform_alibaba")
+load("//tensorflow/compiler/mlir/disc:disc.bzl", "if_platform_alibaba", "if_with_tf_blade", "if_without_tf_blade")
```

Comment:
>  "if_with_tf_blade", "if_without_tf_blade"

The two functions are duplicated? `if_without_tf_blade` is the false branch of `if_with_tf_blade`.

Reply:
They are duplicated.
1. for no-tf_blade build:
use **if_without_tf_blade** for tao_compiler_main's version.h which is generated by cmake and copy to certain dir.  2. for build with tf blade:
version.h is generated by bazel genrule, we don't have a version.h from as a src files. **if_with_tf_blade** is used for  dependency with generated version.h as a cc_library.